ExtraHop/extrahop/reveal_360_trigger.py

```diff
@@ -68,7 +69,7 @@ def fetch_page(self, from_time: int, offset: int) -> list[dict] | None:
             "Content-Type": "application/json",
             "Accept": "application/json",
         }
-        url = f"{self.module.configuration.base_url}/api/v1/detections/search"
+        url = urljoin(self.module.configuration.base_url.lstrip("/"), "api/v1/detections/search")
```

Comment:
I believe it should be `rstrip`, because we remove a slash in the end of URL

Reply:
Now I think that I didn't make this foolproofed enough before, because user can add just a hostname without schema or use an insecure "http://". Perhaps, we could use something like:
```
base_url  = "https://%s" % self.module.configuration.base_url.lstrip("https://").lstrip("http://")
```